marimo/_server/export/exporter.py

```diff
@@ -84,10 +84,7 @@ def export_as_html(
                     e,
                 )
                 continue
-            mime_type, _ = mimetypes.guess_type(basename) or (
-                "text/plain",
-                None,
-            )
+            mime_type = mimetypes.guess_type(basename)[0] or "text/plain"
```

Comment:
NB
 > Deprecated since version 3.13: Passing a file path instead of URL is [soft deprecated](https://docs.python.org/3/glossary.html#term-soft-deprecated). Use [guess_file_type()](https://docs.python.org/3/library/mimetypes.html#mimetypes.guess_file_type) for this.
 
Fine here, but I think we use it in a few places.

Reply:
guess_file_type says added in 3.13